src/app/BreadcrumbPage/BreadcrumbPage.tsx

```diff
@@ -52,7 +60,7 @@ export interface BreadcrumbTrail {
 
 export const BreadcrumbPage: React.FC<BreadcrumbPageProps> = (props) => {
   return (
-    <>
+    <PageGroup>
       <PageSection>
         <Breadcrumb>
```

Comment:
Wrap this in `PageBreadCrumb`?

See: https://www.patternfly.org/v4/components/page/#group-section